src/main/java/cc/hyperium/config/DefaultConfig.java

```diff
@@ -18,6 +18,9 @@
 package cc.hyperium.config;
 
 
+import cc.hyperium.config.handlers.PostConfigHandler;
+import cc.hyperium.config.handlers.PreConfigHandler;
```

Comment:
Whats this about?

Reply:
moved the config handlers to that package